### ISSUE TITLE
Be an EvaluatableExpressionProvider

### ISSUE THREAD

**Issue by gjsjohnmurray** [intersystems/language-server#74]
See docs [here](https://code.visualstudio.com/api/references/vscode-api#EvaluatableExpressionProvider).

Will resolve https://github.com/intersystems-community/vscode-objectscript/issues/486

**Comment by isc-bsaviano** [comment]
I don't think this is part of the LSP: https://microsoft.github.io/language-server-protocol/specifications/specification-current/

**Comment by gjsjohnmurray** [comment]
No, but couldn't your extension activate() register an instance of a class you write that implements vscode.EvaluatableExpressionProvider and uses custom messages to delegate the work to your language server?

**Comment by isc-bsaviano** [comment]
Yes, that would be a possibility.
